src/test/java/org/takes/rq/RqLengthAwareTest.java

```diff
@@ -62,11 +61,12 @@ public void addsLengthToBody() throws IOException {
     }
 
     /**
-     * RqLengthAware can add BIG length to body.
+     * RqLengthAware can respect the real body length over
+     * "Content-Length" value.
      * @throws IOException If some problem inside
      */
     @Test
-    public void addsBigLengthToBody() throws IOException {
+    public void respectRealBodyLength() throws IOException {
```

Comment:
@rui-castro it is misspelled, it should rather be `respectsRealBodyLength` with a s